snapshot/src/test/java/io/zeebe/snapshots/impl/SnapshotChunkReaderTest.java

```diff
@@ -136,78 +125,44 @@ public void shouldSeekToChunk() throws Exception {
     assertThat(snapshotChunks).hasSize(2);
 
     assertThat(snapshotChunkIds).containsExactly(asByteBuffer("file2"), asByteBuffer("file3"));
-
-    final var expectedSnapshotChecksum = SnapshotChecksum.calculate(persistedSnapshot.getPath());
-
-    // chunks should always read in order
-    assertSnapshotChunk(
-        expectedSnapshotChecksum, snapshotChunks.get(0), "file2", "is", persistedSnapshot.getId());
-    assertSnapshotChunk(
-        expectedSnapshotChecksum,
-        snapshotChunks.get(1),
-        "file3",
-        "content",
-        persistedSnapshot.getId());
   }
 
   @Test
   public void shouldThrowExceptionOnReachingLimit() {
     // given
-    final var index = 1L;
-    final var term = 0L;
-
-    final var transientSnapshot =
-        persistedSnapshotStore.newTransientSnapshot(index, term, 1, 0).orElseThrow();
-    transientSnapshot.take(
-        p -> takeSnapshot(p, List.of("file3", "file1", "file2"), List.of("content", "this", "is")));
-    final var persistedSnapshot = transientSnapshot.persist().join();
-
-    // when
     final var snapshotChunkReader = persistedSnapshot.newChunkReader();
     while (snapshotChunkReader.hasNext()) {
       snapshotChunkReader.next();
     }
 
-    // then
     assertThat(snapshotChunkReader.nextId()).isNull();
 
+    // when - then
     assertThatThrownBy(snapshotChunkReader::next).isInstanceOf(NoSuchElementException.class);
   }
 
-  private void assertSnapshotChunk(
-      final long expectedSnapshotChecksum,
-      final SnapshotChunk snapshotChunk,
-      final String fileName,
-      final String chunkContent,
-      final String snapshotId) {
-    assertThat(snapshotChunk.getSnapshotId()).isEqualTo(snapshotId);
-    assertThat(snapshotChunk.getChunkName()).isEqualTo(fileName);
-    assertThat(snapshotChunk.getContent()).isEqualTo(chunkContent.getBytes());
-    assertThat(snapshotChunk.getTotalCount()).isEqualTo(4);
-    final var crc32c = new CRC32C();
-    crc32c.update(asByteBuffer(chunkContent));
-    assertThat(snapshotChunk.getChecksum()).isEqualTo(crc32c.getValue());
-
-    assertThat(snapshotChunk.getSnapshotChecksum()).isEqualTo(expectedSnapshotChecksum);
-  }
-
   private ByteBuffer asByteBuffer(final String string) {
     return ByteBuffer.wrap(string.getBytes()).order(Protocol.ENDIANNESS);
   }
 
-  private boolean takeSnapshot(
-      final Path path, final List<String> fileNames, final List<String> fileContents) {
-    assertThat(fileNames).hasSameSizeAs(fileContents);
+  private long getChunkChecksum(final byte[] content) {
+    final var crc32c = new CRC32C();
+    crc32c.update(content);
+    return crc32c.getValue();
```

Comment:
Can we avoid duplicating the checksum logic here? Could we use `SnapshotChunkUtil#createChecksum` here?